### DISCUSSION TITLE
Lista nadchodzących sprawdzianów i zadań domowych

### DISCUSSION THREAD

**Discussion by artgru** [Czapla-dev/vulcan-for-hassio#55] [Ideas]
Witajcie!

Zauważyłem, że istnieją dwa sensory: najbliższego sprawdzianu i zadania domowego, co jest w sumie ok. 
Ale znając nasze realia szkolne, takich wpisów w ciągu tygodnia się zbiera i obecnie intergracja nie dostarcza żadnej listy np na kolejne X dni. Dzięki temu mógłbym zachować przegląd  nadchodzących wydarzeń związanych ze sprawdzianami czy z.dom a nie tylko najbliższego.

Czy istnieje szansa na wprowadzenie sensora z listą nadchodzących wydarzeń, nawet w formie kalendarza co w połączeniu z planem by było dość ciekawą informacją :)

**Comment by Antoni-Czaplicki** [comment]
Już od jakiegoś czasu planowane: https://github.com/users/Antoni-Czaplicki/projects/1

Dzisiaj wrzucę dużą aktualizację właśnie z tymi kalendarzami i wyzwalaczami urządzeń

**Comment by Antoni-Czaplicki** [comment]
https://github.com/Antoni-Czaplicki/vulcan-for-hassio/releases/tag/0.16.0
